### DISCUSSION TITLE
Adding images in comment

### DISCUSSION THREAD

**Discussion by rtpub88** [community/community#147466] [New to GitHub] [answered]
Hi I'm a newbie here, I have recently created a new issue and tried to provide some screenshots images.  However, when I tried to upload the images into the comment it always fail with the message __ inside the comment.  Any idea how I can get around this? Many thanks!

**Comment by HackedRico** [comment]
Hey, 

This is documentation from Github to [Creating ScreenShots](]https://docs.github.com/en/contributing/writing-for-github-docs/creating-screenshots)

It should just work if you copy and paste the image directly onto the issue like below.

![image](https://github.com/user-attachments/assets/e536e765-de25-468a-87b0-267d3f5995c1)


Hope this helps!

**Reply by rtpub88** [reply]
@HackedRico thx for the reply... it turned out that it was the firewall that blocked access to the destination where the images were uploaded to.

**Reply by angeljameshouser-web** [reply]
hey when ever i click the latest bsmanager it sends me to versien1.44.1 instead of 1.5.6 and i have no one else to ask so pls help

**Comment by eric-mxrtin** [comment]
Hi, 

Use this syntax to reference an image from a URL:

`![Alt Text](https://user-images.githubusercontent.com/username/image-id.png)`

Dragging and dropping or copying and pasting should also generate this text.

**Reply by Calorion** [reply]
When I drag and drop an image, the syntax comes out like 

`<img width="807" height="207" alt="Screen Shot 2025-11-09 at 09 11 11" src="https://github.com/user-attachments/assets/ee86e4b6-9491-4728-8e29-c22970fdafa7" />`

**Comment by rtpub88** [comment, marked as answer]
If the destination (Amazon cloud drive) is blocked by firewall, uploading images will not work

**Reply by aljam3h** [reply]
From f8af9a9bf5370504b04d165677e96497e5cc74f0 Mon Sep 17 00:00:00 2001
From: talaq_alotaibi <95488566+aljam3h@users.noreply.github.com>
Date: Wed, 8 Jan 2025 03:56:59 +0300
Subject: [PATCH] Create README.md

"Integration with Accounts
---
 README.md | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 README.md

diff --git a/README.md b/README.md
new file mode 100644
index 0000000..8b13789
--- /dev/null
+++ b/README.md
@@ -0,0 +1 @@
+

**Comment by ntienanh** [comment]
![Uploading LOGO.png…]()

**Comment by tomcomtang** [comment]
You need to use this kind of markdown format, for example:  
![](https://motionless-magenta-koq0oj05nb.edgeone.app/test.png)

For image URLs, you can try using [this](https://pages.edgeone.ai/use-cases/image-to-url) to drag and drop your local images to get a URL.

**Comment by jernybihaglingaro** [comment]
Copy-paste, and pointing to folder path worked well last week. Its not working now. Any suggestions?

**Reply by jernybihaglingaro** [reply]
Turning off Zscaler VPN worked for me.

**Comment by drankinatty** [comment]
I have this issue, but I can't upload images from my local computer anymore. For more than a decade I had no problems providing screen shots for issues, now I just get an html comment inserted in the issue saying, e.g.  the upload of xyz.png failed. I've read through https://github.com/orgs/community/discussions/28947 and that is not my issue. 

For example if I try to upload an image of this comment here, nothing:

But I do get browser errors in Firefox 152, e.g.

```none
Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at https://github-production-user-asset-6210df.s3.amazonaws.com/. (Reason: CORS request did not succeed). Status code: (null).
Uncaught AttachmentUploadError: App assets attachment upload failed: storage request.
    o attachment-upload.ts:63
    setTimeout handler*o attachment-upload.ts:62
    process attachment-upload.ts:104
    d batch-upload.ts:59
    819435 file-attachment.ts:36
    m index.js:96
    attach index.js:164
    p index.js:246
Caused by: error { isTrusted: true, lengthComputable: false, loaded: 0, total: 0, eventPhase: 0, bubbles: false, cancelable: false, returnValue: true, defaultPrevented: false, composed: false }
attachment-upload.ts:63:35
Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at https://api.github.com/_private/browser/errors. (Reason: CORS request did not succeed). Status code: (null).
```

For what it is worth the .png upload attempt was only 42k,

```none
-rw-r--r--.  1 david david   42322 Jun 25 15:56 github-image-upload.png
```

What's going on, what else to check?
